packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -1983,6 +1993,129 @@ describe('TransactionController', () => {
     });
   });
 
+  describe('updateTransactionSendFlowHistory', () => {
+    it('appends sendFlowHistory entries to transaction meta', async () => {
```

Comment:
As this test is technically not appending but providing the initial flow history, should we have an additional test to verify appended to existing entries?

Reply:
That's true, I've added another one to be sure that entries extended succesfully https://github.com/MetaMask/core/pull/1665/commits/39ac598f58cfc7746f2fd9b625c20d5788b5551e